### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by willard** [icetee/remote-ftp#88]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `Remote-FTP` package

```
Package.getStylesheetsPath (C:\Users\Willard\AppData\Local\atom\app-0.194.0\resources\app.asar\src\package.js:430:9)
Package.getStylesheetPaths (C:\Users\Willard\AppData\Local\atom\app-0.194.0\resources\app.asar\src\package.js:441:32)
Package.loadStylesheets (C:\Users\Willard\AppData\Local\atom\app-0.194.0\resources\app.asar\src\package.js:423:38)
<unknown> (C:\Users\Willard\AppData\Local\atom\app-0.194.0\resources\app.asar\src\package.js:167:19)
```

**Comment by samijnih** [comment]
+1

**Comment by grenierdev** [comment]
Merged pull request #98
